client/packages/programs/src/api/hooks/useCreateRnRForm.ts

```diff
@@ -22,14 +24,33 @@ interface RnRFormDraft {
 
 export const useCreateRnRForm = () => {
   const { mutateAsync, isLoading, error } = useCreate();
-
   const [draft, setDraft] = useState<RnRFormDraft>({
     supplier: null,
     program: null,
     schedule: null,
     period: null,
   });
 
+  const { data } = useRnRFormList({
+    sortBy: {
```

Comment:
Ah interesting! I went with this direction based on your other comment, that otherwise you would end up with a long list of periods that you needed to scroll to the bottom of?